vrc-get-gui/locales/zh_hans.json5

```diff
@@ -95,12 +95,18 @@
     "projects:type:avatars": "虚拟形象",
     "projects:type:legacy": "旧版",
     "projects:last modified:moments": "刚才",
-    "projects:last modified:minutes": "{{count}} 分钟前",
-    "projects:last modified:hours": "{{count}} 小时前",
-    "projects:last modified:days": "{{count}} 天前",
-    "projects:last modified:weeks": "{{count}} 周前",
-    "projects:last modified:months": "{{count}} 月前",
-    "projects:last modified:years": "{{count}} 年前",
+    "projects:last modified:minutes_one": "{{count}} 分钟前",
+    "projects:last modified:minutes_other": "{{count}} 分钟前",
+    "projects:last modified:hours_one": "{{count}} 小时前",
+    "projects:last modified:hours_other": "{{count}} 小时前",
+    "projects:last modified:days_one": "{{count}} 天前",
+    "projects:last modified:days_other": "{{count}} 天前",
+    "projects:last modified:weeks_one": "{{count}} 周前",
+    "projects:last modified:weeks_other": "{{count}} 周前",
+    "projects:last modified:months_one": "{{count}} 月前",
+    "projects:last modified:months_other": "{{count}} 月前",
+    "projects:last modified:years_one": "{{count}} 年前",
+    "projects:last modified:years_other": "{{count}} 年前",
```

Comment:
It looks _one and _other are same so `projects:last modified:minutes` is enough.

_one and _other is for some languages that have different localization is needed, like English need 1 minute and 2 minute**s**

(Non blocking)

Reply:
> _one and _other is for some languages that have different localization is needed, like English need 1 minute and 2 minutes  

Inconsistencies in line numbers can cause difficulties in translation, is it feasible to use comments to fill in the blank lines caused by discrepancies?

Reply:
Inconsistencies in line numbers can cause difficulties in translation, is it feasible to use comments to fill in the blank lines caused by discrepancies?

Currently @lonelyicer is the maintainer of the Simplified Chinese so I want to ask lonelyicer for decisition.
Possible options are:
- keep line numbers with blank line
- keep line numbers with comment line
- keep line numbers with duplicating keys with _other and _one
- do not duplicate keys and allow line number difference

Reply:
> Currently @lonelyicer is the maintainer of the Simplified Chinese.

Ok, thanks for reminding me that I was mistaken for a regular contributor. The maintainer's opinion is important.

Reply:
`Keeping line numbers with blank lines` may break the file classification structure.
`Keeping line numbers with comment lines` works, but makes the file look a bit cluttered.
`keep line numbers with duplicating keys with _other and _one` keeps non-essential keys, which can be a little tricky to maintain.
`do not duplicate keys and allow line number difference` may indeed cause some difficulty, but much less for the difficulties mentioned above.

So my suggestion is `do not duplicate keys and allow line number differences`, but `keep line numbers with comment lines` is also acceptable.